apps/frontend/src/libs/components/popup/popup.tsx

```diff
@@ -28,6 +30,14 @@ const Popup: React.FC<Properties> = ({
 		<dialog className={styles["logout-dialog"]} open={isOpen}>
 			<div className={styles["popup-container"]}>
 				<div className={styles["contents-container"]}>
+					{hasCloseIcon && (
+						<button
+							className={styles["close-icon-container"]}
+							onClick={onClose}
+						>
```

Comment:
Why not use existing Button component?

Reply:
Because I need only a clickable icon, like this
![image](https://github.com/user-attachments/assets/eb415aa1-0362-41bf-87f7-a65af13c8c14)

Can I change the properties of the existing Button component to remove styles?

Reply:
You can add a new `variant` or `type` prop, whichever we already have